guide/english/algorithms/binary-search-trees/index.md

```diff
@@ -358,7 +358,7 @@ Let x be any arbitary node of a tree.
 
 x.size = x.left.size + x.right.size + 1
 
-While augumenting the tree, we should keep in mind, that we should be able to maintain the augumented information as well as do other operations like insertion, deletion, updation in O(lg n) time.
+While augumenting the tree, we should keep in mind, that we should be able to maintain the augumented information as well as do other operations like insertion, deletion, updation in `O(log n)` time.
```

Comment:
Errant comma, spelling error, plus 'updation' isn't a word.
"While augumenting the tree, we should keep in mind that we should be able to maintain the augmented information, as well as perform other operations such as insertion, deletion, updating in `O(log n)` time."